### PR TITLE
Import common product from Akeneo as Sylius Product

### PR DESCRIPTION
**Pull request by TheGrimmChester**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?   | yes

### CODE REVIEW

**Review by tuyetrinhvo — APPROVED** [review]

File: .github/workflows/sylius.yaml [hunk 1]
```diff
@@ -164,7 +164,6 @@ jobs:
             -
                 name: 'Run PHPUnit'
                 run: 'make phpunit'
-                if: 'always() && steps.end-of-setup-sylius.outcome == ''success'''
```
Comment (oallain) [inline .github/workflows/sylius.yaml:167]: You need to do the same for "Doctrine Schema Validate" line 163.

File: src/Builder/ProductOptionValueTranslation/MetricProductOptionValueTranslationBuilder.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    ): ProductOptionValueTranslationInterface {
+        $attributeCode = $productOption->getCode();
+        Assert::string($attributeCode);
+
+        /** @var array{unit: string, amount: string} $data */
+        $data = $this->akeneoAttributeDataProvider->getData(
+            $attributeCode,
+            $attributeValues,
+            $locale,
+            'ecommerce'
```
Comment (oallain) [inline src/Builder/ProductOptionValueTranslation/MetricProductOptionValueTranslationBuilder.php:85]: You need to use ApiConnectionProvider to get the scope.

File: src/Builder/ProductOptionValueTranslation/MetricProductOptionValueTranslationBuilder.php [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        );
+
+        $productOptionValue->setCurrentLocale($locale);
+
+        $properties = $this->akeneoAttributePropertiesProvider->getProperties($attributeCode);
+        $measurable = $this->measureFamilyRetriever->getMeasurable($properties['metric_family'], $data['unit']);
+        $productOptionValue->setValue(\sprintf('%s %s', (float) $data['amount'], $measurable['symbol']));
```
Comment (oallain) [inline src/Builder/ProductOptionValueTranslation/MetricProductOptionValueTranslationBuilder.php:92]:
```suggestion
        $productOptionValue->setValue(\sprintf('%f %s', (float) $data['amount'], $measurable['symbol']));
```

**Review by oallain — CHANGES_REQUESTED** [review]
It's a good job, I just put some comments.

**Review by oallain — APPROVED** [review]

**Pull request merged**